src/html/template.html

```diff
@@ -1,4 +1,4 @@
-       <div class="fl-mapviz-outer-container">
+        <div class="fl-mapviz-outer-container">
```

Comment:
Is this the same wrapper container in `index.html`? Shall this line be removed from here?